src/PantheonTerminalWindow.vala

```diff
@@ -251,20 +251,33 @@ namespace PantheonTerminal {
 
             search_button = new Gtk.ToggleButton ();
             search_button.image = new Gtk.Image.from_icon_name ("edit-find-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
-            search_button.tooltip_text = _("Find…");
             search_button.valign = Gtk.Align.CENTER;
+            search_button.tooltip_markup = Granite.markup_accel_tooltip (
+                {"<Ctrl><Shift>f"},
+                _("Find…")
+            );
+
 
             var zoom_out_button = new Gtk.Button.from_icon_name ("zoom-out-symbolic", Gtk.IconSize.MENU);
-            zoom_out_button.tooltip_text = _("Zoom Out");
             zoom_out_button.action_name = ACTION_PREFIX + ACTION_ZOOM_OUT_FONT;
+            zoom_out_button.tooltip_markup = Granite.markup_accel_tooltip (
+                application.get_accels_for_action (zoom_out_button.action_name),
+                _("Zoom Out")
+            );
 
             zoom_default_button = new Gtk.Button.with_label ("100%");
-            zoom_default_button.tooltip_text = _("Default zoom level");
             zoom_default_button.action_name = ACTION_PREFIX + ACTION_ZOOM_DEFAULT_FONT;
+            zoom_default_button.tooltip_markup = Granite.markup_accel_tooltip (
+                application.get_accels_for_action (zoom_default_button.action_name),
+                _("Default zoom level")
+            );
 
             var zoom_in_button = new Gtk.Button.from_icon_name ("zoom-in-symbolic", Gtk.IconSize.MENU);
-            zoom_in_button.tooltip_text = _("Zoom In");
             zoom_in_button.action_name = ACTION_PREFIX + ACTION_ZOOM_IN_FONT;
+            zoom_in_button.tooltip_markup = Granite.markup_accel_tooltip (
+                application.get_accels_for_action (zoom_in_button.action_name),
+                _("Zoom In")
```

Comment:
So we were bad about this in Code, but I guess this is supposed to be sentence case? https://elementary.io/docs/human-interface-guidelines#text-buttons-tooltips